lib/presentation/organisms/intro/tips_organism.dart

```diff
@@ -37,6 +38,24 @@ class TipsOrganism extends StatelessWidget {
             translate: false,
           ),
           const SeparatorAtom(variant: SeparatorVariant.farApart),
+          Center(
+              child: ElevatedButton(
+                  onPressed: () async {
+                    bool filter =
```

Comment:
The linked plugin is old, deprecated and fails static analysis so I don't think we should use it to implement DND functionality for windows

Reply:
It doesn't matter if we use the windows one or not.
Architecture of this project is built so that logic and outside API calls (even to the device) should be placed under infrastructure folder.

Reply:
Yes, That is understandable, all business logic was moved to infrastructure folder